### ISSUE TITLE
tidy3d module import side effects

### ISSUE THREAD

**Issue by joamatab** [flexcompute/tidy3d#650]
when importing tidy3 module there are two strange side effects that I don't really want and seem unnecessary

1. the stdout format changes. This causes plots to be duplicated, and makes it incompatible with other packages.
2. the logging level changes to everything so I can see logging even INFO messages

See this [notebook](https://gdsfactory.github.io/ubc/notebooks/11_sparameters.html)

I think is happening in the `__init__.py`
```
# make all stdout and errors pretty
pretty.install()
# traceback.install()
log.info(f"Using client version: {__version__}")
```

@tylerflex 
@momchil-flex

**Comment by tylerflex** [comment]
@joamatab I'm having trouble reproducing this issue. I made a branch off of develop to try to come up with some test, but it seems that importing tidy3d is not changing the logging level of either the root or a custom logger. https://github.com/flexcompute/tidy3d/blob/fix/log_level_reset/tests/test_package/test_log.py#L33-L47

I'm not as familiar with the logging module as you might be so maybe I'm doing something dumb here but if you have any ideas on how to reproduce this feel free to suggest.

**Comment by joamatab** [comment]
Hi Tyler,

the issue is coming from here
![image](https://user-images.githubusercontent.com/4514346/212563357-d2601f62-b7f4-449f-9516-c0a06ebd46ed.png)


my workaround to setting the default level of your logger to INFO is using this code in gdsfactory, where I explicit set your rich logger to `WARNING`

```
import sys
import logging
from rich.logging import RichHandler

logger = logging.getLogger()
logger.removeHandler(sys.stderr)
logging.basicConfig(level='WARNING', datefmt="[%X]", handlers=[RichHandler()])
``

**Comment by tylerflex** [comment]
Thanks Joaquin, how is your logger initialized in gdsfactory?
If I can initialize it the same way as you [in this test](https://github.com/flexcompute/tidy3d/blob/23661fbae9d816cca2d79eaee6c3a34bc87aab1e/tests/test_package/test_log.py#L33-L47) then it would help me isolate the issue. As you can see in the test, for some reason I am not able to observe importing tidy3d resetting the level of the logger initialized in a couple different ways. so if you can point me in the right direction I can take a look.

**Comment by momchil-flex** [comment]
@tylerflex I suspect this may be interfering with other packages that use the rich logger specifically.

**Comment by tylerflex** [comment]
Hm, yea tried changing the test to use a rich logger though and importing tidy3d still didn't seem to affect the log level.
https://github.com/flexcompute/tidy3d/blob/fix/log_level_reset/tests/test_package/test_log.py#L39-L57

**Comment by tylerflex** [comment]
Actually, I did realize just now that initializing this logger seems to change the tidy3d log level, so that might be the smoking gun. Will look into it a bit more this afternoon.

**Comment by tylerflex** [comment]
Hey @joamatab can you try tidy3d from [this branch](https://github.com/flexcompute/tidy3d/pull/663) and see if it's fixed?

If not, could you tell me how you initialize your logger in gdsfactory? It seems you use `log=logger.root` somewhere? I'm having trouble isolating the issue to be honest, but did manage to find a test that seemed to be related to naming conflicts in the `logging.getLogger` call. For example, if we both call our logger "rich", they will be coupled..

**Comment by joamatab** [comment]
Hi Tyler,

it's not fixed, you can use this code to reproduce the issue

```
import logging
import tidy3d as td

LOGGER_NAME = "my_logger"
logger = logging.getLogger(LOGGER_NAME)

logger.info("i do not want to show info logs by default")
```

**Comment by joamatab** [comment]
looking forward to a new release that solves this issue

https://github.com/gdsfactory/gdsfactory/issues/1330

**Comment by tylerflex** [comment]
It's on my to do list, we just have a lot of other priorities at the moment. I should get to it this week or early next week. thanks for your patience.

**Comment by tylerflex** [comment]
@lucas-flexcompute do you have any idea how to fix this? it stumped me but I feel like maybe you'd know.

**Comment by lucas-flexcompute** [comment]
I've worked with the logging module before. I'll take a look, as soon as I can!

**Comment by lucas-flexcompute** [comment]
@tylerflex, @joamatab I've changed the internal log module to not modify the global logging configuration. I've also changed the level target: it is now applied to the handler, not the logger, so that the rich and file handlers can have truly independent levels.

**Comment by tylerflex** [comment]
@lucas-flexcompute did you make a PR for this yet? I'm not seeing anything. Thanks a lot for taking care of this, apparently I didn't set up the logging level quite correctly initially, this makes a lot of sense.

**Comment by lucas-flexcompute** [comment]
I have just [created it](https://github.com/flexcompute/tidy3d/pull/727). I was incorporating the other changes I mentioned elsewhere. I've left the partial commits without squashing in case we don't want all the modifications.

**Comment by tylerflex** [comment]
@joamatab I just released a patch 1.9.2 with our own custom logger, written by Lucas. It should fix this issue, feel free to try it out and let me know how it goes!
